src/new-client/src/App/Analytics/AnalyticsCore.tsx

```diff
@@ -7,10 +7,12 @@ import {
   AnalyticsStyle,
   AnalyticsHeader,
   AnalyticsInnerWrapper,
+  RightNav,
 } from "./styled"
 import { createSuspenseOnStale } from "@/utils/createSuspenseOnStale"
 import { isAnalyticsDataStale$ } from "@/services/analytics"
-
+import { supportsTearOut } from "@/Web/TearOutSection/supportsTearOut"
```

Comment:
This shouldn't live in `@Web` as it has different platform implementations.

Reply:
Fixed, I didn't know where to put it so for now you can find it  at `@/App/supportsTearOut`